doc/site/en/guides/border-router/beaglebone-black.md

```diff
@@ -152,10 +152,10 @@ with the command `sudo vim /boot/uEnv.txt` and make sure the following lines
 match:
 
 ```
-###Custom Cape
+#Custom Cape
```

Comment:
This won't match what is in the file `/boot/uEnv.txt`, but I think people will understand anyway.

Reply:
I would hope so, if it's a problem we can address later.